scripts/e2e-ci.sh

```diff
@@ -210,8 +210,14 @@ test_taxonomy() {
 }
 
 test_generate() {
-    task Generate synthetic data
-    ilab data generate
+    task Generate synthetic data in detached mode
+    ilab data generate -dt
+    task Data Generation started
```

Comment:
Capitalization should be consistent with the rest of the script
```suggestion
    task Data generation started
```